projects/ngx-charts-on-fhir/src/lib/data-layer-toolbar/data-layer-toolbar/data-layer-toolbar.component.css

```diff
@@ -3,6 +3,7 @@
   background-color: #3f51b5;
   position: absolute;
   height: 100vh;
-  width: 48px;
+  width: 52px;
   padding-top: 16px;
+  z-index: 999;
```

Comment:
Changing `z-index` is not the best solution. It's better to avoid overlapping elements in the first place. Please change the `left` property of `mat-sidenav-container` in `app.component.scss` to match the toolbar width.